src/Utils/Logging/Config.hs

```diff
@@ -130,19 +134,27 @@ handleConfigFromText x = case CI.mk x of
     "stdout" -> return StdOut
     "stderr" -> return StdErr
     _ | CI.mk (T.take 5 x) == "file:" -> return $ FileHandle (T.unpack (T.drop 5 x))
-    _ | CI.mk (T.take 3 x) == "es:" -> return $ ElasticSearch (T.drop 3 x)
+    _ | CI.mk (T.take 4 x) == "es::" -> return $ ElasticSearch (T.drop 4 x) Nothing
+    _ | CI.mk (T.take 7 x) == "es:http" -> return $ ElasticSearch (T.drop 3 x) Nothing
+        -- legathy format without API key (deprecated)
```

Comment:
```suggestion
        -- legacy format without API key (deprecated)
```

Reply:
Also it's unclear which case this refers to. Assuming it's the above, the comment should probably go above